src/modules/Routes/NewBook.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function NewBook() {
+  return (
+    <div>
+      <div className="container">
+        <h3>ADD NEW BOOK</h3>
+        <form>
+          <input placeholder="Book Title" />
+          <input placeholder="Book Author" />
+        </form>
+        <button type="button">Add book</button>
```

Comment:
✅  Renders a form with inputs for the title and the author of the book 👍🏻 
✅  Has a button to submit 👍🏻

Reply:
Here is a form with all requirements